MonMaraicher/DetailFarmerView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailFarmerView.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 27/02/2024.
+//
+
+import SwiftUI
+import MapKit
+
+struct DetailFarmerView: View {
+    
+    @State var selectedFarmer: FarmerPlace?
+    
+    @Binding var show: Bool
+    
+    var body: some View {
+        
+        Text(selectedFarmer?.name ?? "")
+            .padding()
+        Button {
```

Comment:
There is a pb here with your view hierarchy. You put a Text and a Button. You probably forgot to put them inside a VStack or HStack right?

Reply:
It was a test content, the view will be much richer than that